src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -1560,6 +1560,44 @@ def test_webapp_create_linux_free(self, resource_group):
         self.cmd('webapp config show -g {} -n {}'.format(resource_group, webapp_name)).assert_with_checks([
             JMESPathCheck('alwaysOn', False)])
 
+    @ResourceGroupPreparer(name_prefix='cli_test_webapp_linux_free')
+    def test_webapp_create_linux_free(self, resource_group):
+        webapp_name = self.create_random_name('webapp-linux-free', 24)
+        plan = self.create_random_name('plan-linux-free', 24)
+
+        self.cmd('appservice plan create -g {} -n {} --sku F1 --is-linux'.format(resource_group, plan), checks=[
+            JMESPathCheck('reserved', True),  # this weird field means it is a linux
+            JMESPathCheck('sku.name', 'F1')])
+        self.cmd('webapp create -g {} -n {} --plan {} -u {} -r "node|10.14"'.format(resource_group, webapp_name, plan,
+                                                                                    TEST_REPO_URL))
+        # verify alwaysOn
+        self.cmd('webapp config show -g {} -n {}'.format(resource_group, webapp_name)).assert_with_checks([
+            JMESPathCheck('alwaysOn', False)])
+
+
+class WebappHybridConnectionTest(ScenarioTest):
+    @ResourceGroupPreparer()
+    def test_webapp_hybridconnection(self, resource_group):
+        webapp_name = self.create_random_name(prefix='webapp-hc', length=24)
+        namespace_name = self.create_random_name(prefix='namespace-hc', length=24)
+        # hc_name = self.create_random_name(prefix='hc', length=24)
+        plan = self.create_random_name('plan-hc', 44)
+        um = "[{{\"key\":\"endpoint\",\"value\":\"vmsq1:80\"}}]"
+
+        self.cmd('appservice plan create -g {} -n {}'.format(resource_group, plan))
+        self.cmd('webapp create -g {} -n {} -p {}'.format(resource_group, webapp_name, plan))
+        self.cmd('relay namespace create -g {} --name {} --location westus'.format(resource_group, namespace_name))
+        self.cmd('relay hyco create -g hybridConnectionTest --namespace-name myCompTest --name test-hcRelay --user-metadata {} --debug'.format(um))
+        # self.cmd('webapp hybrid-connection list -g {} -n {}'.format(resource_group, webapp_name), checks=[
+        #     JMESPathCheck('length(@)', 0),
+        # ])
+        # self.cmd('webapp hybrid-connection add -g {} -n {} --namespace {} --hybrid-connection {}'.format(resource_group, webapp_name, namespace_name, hc_name))
```

Comment:
I assume you will finish up the test here, right?

Reply:
@yugangw-msft we have an email thread on this for help getting the test to work - we are waiting for some response from your team on this to close on this one.

Reply:
@adewaleo we are still have trouble getting this test to work here can we set-up sometime with you for later this week to close on this please?